algorithms/CSharp/src/Search/minima-maxima.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Algorithms.Search
+{
+    public class MinimaMaxima
+    {
+        public static void Main()
+        {
+            var numbers = new List<int>
+            {
+                3,1,2,5,6,7,4,6,9,10
+            };
+
+            var lmins = GetLocalMinima(numbers);
+            var lmaxs = GetLocalMaxima(numbers);
+
+            foreach (var n in lmins)
+                Console.WriteLine($"Local Minima: {n}");
+
+            foreach (var n in lmaxs)
+                Console.WriteLine($"Local Maxima: {n}");
```

Comment:
Add missing curly brackets
```suggestion
            foreach (var n in lmins)
            {
                Console.WriteLine($"Local Minima: {n}");
            }

            foreach (var n in lmaxs)
            {
                Console.WriteLine($"Local Maxima: {n}");
            }
```